core/src/main/java/io/seata/core/rpc/netty/NettyClientChannelManager.java

```diff
@@ -165,9 +169,19 @@ void reconnect(String transactionServiceGroup) {
             return;
         }
         if (CollectionUtils.isEmpty(availList)) {
-            String serviceGroup = RegistryFactory.getInstance()
-                                                 .getServiceGroup(transactionServiceGroup);
-            LOGGER.error("no available service '{}' found, please make sure registry config correct", serviceGroup);
+            RegistryService registryService = RegistryFactory.getInstance();
+            String clusterName = registryService.getServiceGroup(transactionServiceGroup);
+
+            if (StringUtils.isBlank(clusterName)) {
+                LOGGER.error("can not get cluster name in registry config '{}{}', please make sure registry config correct",
+                        ConfigurationKeys.SERVICE_GROUP_MAPPING_PREFIX,
+                        transactionServiceGroup);
+                return;
+            }
+
+            if (!(registryService instanceof FileRegistryServiceImpl)) {
+                LOGGER.error("no available service found, please make sure registry config correct and keep your seata server running");
```

Comment:
Is it better to prompt the cluster name here?

Reply:
> Is it better to prompt the cluster name here?

fixed,please review it